src/components/Switch/index.tsx

```diff
@@ -94,12 +112,18 @@ const StyledSwitch = styled('div', {
   border-radius: 34px;
   position: relative;
   transition: all 250ms;
-  cursor: pointer;
```

Comment:
Shouldn't we have a `cursor: pointer` on the whole switch? And `cursor: not-allowed` when disabled

Reply:
i'm agree with @QuiiBz, it's weird when you try to click.

https://user-images.githubusercontent.com/14060273/147454947-e4162516-8c63-49e4-a27a-7bf53d26bae4.mov

Reply:
Vu avec Dodo, non necessaire le cursor pointer est deja la.